source/_posts/2020-01-15-release-104.markdown

```diff
@@ -74,28 +74,135 @@ Experiencing issues introduced by this release? Please report them in our [issue
+- __Keba__ - The unique_ids of the entities have changed and are now more future proof. The default name and entity_id of the entities have also changed. Users need to clear stale entries from the entity registry and update concerned automations and scripts. - ([@dannerph] - [#30125]) ([keba docs]) 
+- __Tesla__ - Config flow has been added for Tesla. Device tracker entity names have changed due to the conversion to config_flow. They now follow the naming convention of the rest of the Tesla entities. The `id` device tracker entity attribute was also removed because it is a duplicate of `trackr_id`. - ([@alandtse] - [#28744]) ([tesla docs]) 
+- __Huawei LTE__ - The Huawei LTE notify service name is now configurable. The default name for the Huawei LTE notify service has changed to `notify.huawei_lte` to avoid conflicts with other notify services. An option to provide a name for the service has also been added. - ([@scop] - [#30208]) ([huawei_lte docs])
+
+   Example configuration:
+   ```yaml
+   huawei_lte:
+     - url: ...
+       password: ...
+       notify:
+         name: sms
+   ```
+
+- __Amcrest__ *(Removed)* - Amcrest sensors and switches that were deprecated in previous releases have now been removed. - ([@pnbruckner] - [#30308]) ([amcrest docs])
+
+   Example configuration:
+   ```yaml
+   amcrest:
+   - ...
+     sensors:
+       # This sensor type is no longer supported and must be removed.
+       - motion_detector
+     # All switch types are no longer supported and must be removed.
+     switches:
+       - motion_detection
+       - motion_recording
+   ```
+- __MQTT__ - Add separate command and state topics for mqtt lock. Two new properties were introduced with `state_locked` (default: LOCKED) and `state_unlocked` (default: UNLOCKED). If you were using the same topic as `state_topic` as for the `command_topic` you can reestablish the old behaviour by setting `state_locked` to *LOCK* and `state_unlocked` to *UNLOCK*. Effectively this change allows you to distinguish commands and states in two topics with different values. - ([@tuxbox] - [#29808]) ([mqtt docs])
+
+   Example configuration:
+   ```yaml
+   lock:
+     - platform: mqtt
+       command_topic: "/dev/ha/lock/cmd"
+       state_topic:   "/dev/ha/lock/state"
+       value_template: "{{ value_json.state }}"
+       state_locked: "locked"
+       state_unlocked: "unlocked"
+   ```
+- __LG webOS Smart TV__
+  - Migrate webostv to new library and make integration async with callback state updates. Since the connection handling has become non-trivial, it is now moved to `init.py` to be shared between the media_player and notify platforms, so the component setup now has to be done from the webostv domain instead of in media_player. The host configuration option is now required (as auto-discovery was anyways not functional) and the timeout and filename options have been removed as there is no real need to change the defaults. - ([@bendavid] - [#29296]) ([webostv docs])
```

Comment:
What does this extra dash do here?

Reply:
This merges the two separate prs under webOS heading into bullet points? That's how I've been doing this in previous posts.